ipatests/test_integration/test_installation.py

```diff
@@ -165,13 +168,15 @@ def test_replica0_ipa_kra_install(self):
         tasks.install_kra(self.replicas[0], first_instance=False)
 
 
+@pytest.mark.xfail(reason="FreeIPA ticket 7008")
```

Comment:
Here installs only master. According to [7008](https://pagure.io/freeipa/issue/7008) should not be an issue.